tests/dnp3-dnp3_obj-alert/suricata.yaml

```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-3.1.2.yaml
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular
+      filename: eve.json
+      types:
+        - alert:
+            payload: yes
+            packet: yes
+            dnp3: yes
```

Comment:
This looks way different than what we have in `suricata.yaml` now. Perhaps this kind of structure was followed in old revs? ref: https://github.com/OISF/suricata/blob/master/suricata.yaml.in#L156

Reply:
yes, I just followed the format that was in the previous test. Do I change it for all the dnp3 tests or just the ones I updated.

Reply:
Perhaps, just this one for now. Thanks! :)